pkg/abaputils/manageGitRepositoryUtils_test.go

```diff
@@ -0,0 +1,157 @@
+package abaputils
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPollEntityPullStep(t *testing.T) {
```

Comment:
We should keep the tests in magageGitRepositoryUtils independent from steps.
But, is there a real dependecy to "pull" or "checkout" or is this only the naming?

Reply:
The difference is a check for only one or multiple repositories, see: 

`config := AbapEnvironmentCheckoutBranchOptions{
			AbapEnvOptions: options,
			RepositoryName: "testRepo1",
		}` 
and 
`config := AbapEnvironmentPullGitRepoOptions{
			AbapEnvOptions:  options,
			RepositoryNames: []string{"testRepo1", "testRepo2"},
		}`

Reply:
As far as I can see, this makes no difference for the function to be tested as only a single string is passed to the function:
`status, _ := PollEntity(config.RepositoryNames[0], con, client, 0)`
`status, _ := PollEntity(config.RepositoryName, con, client, 0)`

Reply:
Yes you are right. I adapted the code accordingly.